src/me/eglp/gv2/commands/fun/CommandReminder.java

```diff
@@ -0,0 +1,165 @@
+package me.eglp.gv2.commands.fun;
+
+import java.awt.Color;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import me.eglp.gv2.util.base.guild.GraphiteGuildMessageChannel;
+import me.eglp.gv2.util.base.guild.GraphiteModule;
+import me.eglp.gv2.util.base.guild.reminder.GuildReminder;
+import me.eglp.gv2.util.base.guild.reminder.ReminderRepetitionEnum;
+import me.eglp.gv2.util.command.Command;
+import me.eglp.gv2.util.command.CommandCategory;
+import me.eglp.gv2.util.command.CommandInvokedEvent;
+import me.eglp.gv2.util.command.ParentCommand;
+import me.eglp.gv2.util.lang.DefaultLocaleString;
+import me.eglp.gv2.util.lang.DefaultMessage;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.channel.ChannelType;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.commands.build.OptionData;
+
+/**
+ * This is the reminder Command class. It handles the different sub-commands and
+ * deligates the functionality to {@link GuildReminder}.
+ * 
+ * @author The Arrayser
+ * @date Mon Mar 27 18:28:48 2023
+ */
+
+public class CommandReminder extends ParentCommand {
+
+	public CommandReminder() {
+		super(GraphiteModule.FUN, CommandCategory.FUN, "reminder");
+		setDescription(DefaultLocaleString.COMMAND_REMINDER_DESCRIPTION);
+
+		addSubCommand(new Command(this, "create") {
+
+			@Override
+			public void action(CommandInvokedEvent event) {
+				// "{prefix}reminder create <date and time> <reminder message> [repeat (1y2d)]
```

Comment:
Leftover comment

Reply:
How else would a programmer, that cannot execute the code at the moment - or ever, figure out, how that command is supposed to work, and then compare it to the actual implementation, to find potential bugs and issues, and maybe restructure, or rewrite the code partially. Therefor this is not a "leftover comment", but rather a "comment" (that's what comments are for... commenting (part of) the code, and making maintanance easier...)

Reply:
They can:
- Read the declaration of the options at the bottom, which is always up to date
- Read the current usage message of the command

Reply:
standalone code, this file only, printed out on a papersheet -> this code getting discused in a public scenario, where no computers are available to research the accompaning source code or use discordo...

Reply:
You are proposing very unlikely scenarios

Reply:
What else are comments used for if not for this elite example textbook case...